### PR TITLE
TODO: challenge tutorial

### PR DESCRIPTION
**Pull request by luke-song**

### CODE REVIEW

File: docs/tutorial/gateway-protocol [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+3. Return a CryptidClient object that is built using the Cryptid class and the buildFromDID method: pass in the DID identifier, authority wallet, and middleware programs as arguments.
+4. Use the setupCryptid function by passing in a Wallet object for the authority parameter and a Connection object for the connection parameter to initialize a CryptidClient object.
+
+This will return a Promise that will resolve to a CryptidClient object.
+
+Third, we will propose and execute the transaction.
+
+```ts
+
+//proposeAndExecute function takes in a Transaction object and a CryptidClient object, and logs the transaction and the proposeTransaction and proposeSigners returned by the cryptid.propose method. It then sends the propose transaction to the network, skipping the preflight check. It then sends the executeTransactions and executeSigners returned by the cryptid.execute method to the network.
```
Comment (JulianSpring) [inline docs/tutorial/gateway-protocol]:
```suggestion
// proposeAndExecute function takes in a Transaction object and a CryptidClient object, and logs the transaction and the proposeTransaction and proposeSigners returned by the cryptid.propose method. It then sends the propose transaction to the network, skipping the preflight check. It then sends the executeTransactions and executeSigners returned by the cryptid.execute method to the network.
```

File: docs/tutorial/gateway-protocol [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+6. You can now use the updated DID with the added service for any operations that require that service.
+
+Once we have done all the steps, we will have a gateway pass issued to the user.
+
+```ts
+
+(async () => {
+  const connection = new Connection(clusterApiUrl(cluster));
+  // const connection = new Connection("https://rough-misty-night.solana-devnet.quiknode.pro/b57300ff234c12e95763e9b8cda67e9d86772a0d/")
```
Comment (JulianSpring) [inline docs/tutorial/gateway-protocol]: I don't think we should have commented out code in the tutorial. I think it is a bit confusing. Some below, with `authority` and the airdrops

File: docs/tutorial/gateway-protocol [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+  console.log('Setting up Cryptid');
+  // Create Cryptid Account that uses middleware
+  const cryptid = await setupCryptid(wallet, connection);
+  console.log(`cryptid DID: ${cryptid.did}`);
+  console.log(`cryptid address: ${cryptid.address()}`);
+
+  // await airdrop(connection, cryptid.address());
+
+  // TODO: Implement here to be able to pass the challenge
```
Comment (JulianSpring) [inline docs/tutorial/gateway-protocol]: Same with this, what do we want to do with the commented out code?

Comment (DiosAkhilleus) [reply docs/tutorial/gateway-protocol]: Will remove these!

**Review by JulianSpring — APPROVED** [review]

**Pull request merged**